app.js

```diff
@@ -1,8 +1,125 @@
 // import functions and grab DOM elements
+import { renderPastPoll } from "./render-utils.js";
+
+const optionForm = document.querySelector('#option-form');
+
+const optionAAddButton = document.querySelector('#add-option-a');
+const optionBAddButton = document.querySelector('#add-option-b');
+const optionAUndoButton = document.querySelector('#undo-option-a');
+const optionBUndoButton = document.querySelector('#undo-option-b');
+
+
+const optionAVotesEl = document.querySelector('#option-b-count');
+const optionBVotesEl = document.querySelector('#option-b-count');
+const optionATitleAEl = document.querySelector('#option-a-title');
+const optionBTitleEl = document.querySelector('#option-b-title');
+
+const questionEl = document.querySelector('#poll-question');
+const closePollEl = document.querySelector('#close-poll');
+
+const pastPollsEl = document.querySelector('#display-past-polls');
+
+
+
 
 // let state
+let question = '';
+let voteA = '';
+let voteB = '';
+let optionA = '';
+let optionB = '';
```

Comment:
I'm a little confused with your state variable names here, and I am guessing that might be part of why things are rendering on the page correctly.  I assume voteA and voteB are the vote counts and optionA and optionB are the names of the options?  If you are incrementing or decrementing something, it needs to be a number. So I would initialize whatever state variable is representing your votes as `0` instead of an empty string.